mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/jobmaster/clutch/ClutchConfiguration.java

```diff
@@ -119,38 +120,38 @@ public boolean equals(Object o) {
         if (Double.compare(this.getRps(), other.getRps()) != 0) return false;
         final Object this$minSamples = this.getMinSamples();
         final Object other$minSamples = other.getMinSamples();
-        if (this$minSamples == null ? other$minSamples != null : !this$minSamples.equals(other$minSamples))
+        if (!Objects.equals(this$minSamples, other$minSamples))
```

Comment:
@sundargates this code is too verbose. I'm wondering if it's a good idea to pull in commons-lang3 dependency and use `EqualsBuilder()` for this.